app/searchApi.rb

```diff
@@ -265,6 +265,23 @@ def normalizeResponse(response)
   end
 end
 
+def getInfoResults(q)
```

Comment:
OK, I know this will all change but since I got this working with the fulltext index I figure I'll go ahead and push it for now.